css/src/components/table.scss

```diff
@@ -0,0 +1,128 @@
+/* stylelint-disable selector-no-qualifying-type */
+
+/* stylelint-disable selector-max-compound-selectors */
+
+/* stylelint-disable no-descending-specificity */
+
+.table {
+	width: 100%;
+	font-size: $font-size-8;
+	table-layout: auto;
+
+	caption {
+		padding: $font-size-6;
+		border-width: 0 0 $table-border-width;
+		border-style: solid;
+		border-color: $border;
+		font-size: $table-caption-size;
+		font-weight: $weight-semibold;
+		letter-spacing: $table-caption-spacing;
+		text-align: $user-right;
+		text-transform: uppercase;
```

Comment:
double checking whether this transform should happen by default or instead use a utility

Reply:
I think it should happen by default. In docs it's done that way and as base styles, designers said we're okay with keeping the same styles.